### ISSUE TITLE
Implementing the RosettaFold model in PaddleHelix

### ISSUE THREAD

**Issue by riversdark** [PaddlePaddle/PaddleHelix#147]
Hi I'm interested in reimplementing the RossettaFold 3D protein structure prediction model in PaddleHelix (the model is written originally in PyTorch), but I'm still quite new with PaddleHelix so I thought that I should ask around a bit before I get down to actually do it.

- Are you guys interested in implementing it yourselves? (Perhaps you have already implemented it somewhere...)
- If not, are you interested in accepting a PR for it?
- I feel that most of the components necessary for implementing the model is already present in PaddleHelix, perhaps someone can point me to the right places?
- Are there any anticipated difficulty for implementing the model?

Basically I want to reimplement the model in PaddleHelix so I think I should discuss it here for some general suggestions. Comments are highly appreciated!

**Comment by SuperXiang** [comment]
Hi there, thanks for your interest. We are open to cooperation and you can PR to the repo. As for RoseTTAFold, I think everything except training part is in the original repo. Please keep following our repo for the latest news and updates.
